Source/Components/ComponentBase.cst

```diff
@@ -4,14 +4,14 @@
  * Last modified at $Date: 2005/06/27 12:12:42 $
  * $Revision: 1.31 $
 
-	Written By:
-		Ryan Hurdon
-	Description:
-		This script will create data access layer classes. It requires stored procedures
-		created from the procedure script. So for this code to work properly you need to run
-		the procedure template, the rowobject template, and the rowcollection template on
-		the same table. You can also optionally use the main script which will run them all
-		for you.
+    Written By:

```

Comment:
Would be nice if future whitespace changes were just committed to master and left out of pr's :) With that said thanks for fixing the whitespace :)